dashboard/app.R

```diff
@@ -27,7 +27,7 @@ aboutPageText = includeMarkdown("about.md")
 ########
 ui <- fluidPage(padding=0,
     tags$head(
-      tags$link(rel = "stylesheet", type = "text/css", href = "style.css")
+      tags$link(rel = "stylesheet", type = "text/css", href = "style_v1.2.css")
```

Comment:
I figured that until we have a better way of doing this, this would take care of the issue, especially since we already have to update the version number in the description file during the release.

Reply:
It's pretty easy to add this, I opened #92 to show the basic method.  This is basically how cache busting works in many web-frameworks.  The approach is to refer to deployed assets with md5sum hashes in their name so you know for sure you're getting the right file and if the content changes, it'll get a new md5sum.